package.json

```diff
@@ -22,6 +22,7 @@
   "contributes": {},
   "scripts": {
     "vscode:prepublish": "npm run compile",
+    "vscode:publish": "vsce publish",
```

Comment:
How the `vsce` gets into remote `ubuntu/latest` environment? Is the tool included in `actions/setup-node@v4`?